src/zone/scheduler.h

```diff
@@ -32,9 +32,11 @@ namespace zone {
         /// <summary> Constructor. </summary>
         /// <param name="settings"> A settings object. </param>
         /// <param name="workerSetupCallback"> Callback to setup the isolate after worker created its isolate. </param>
+        /// <param name="exitCallback"> Callback to all workers exit. </param>
         SchedulerImpl(
             const settings::ZoneSettings& settings,
-            std::function<void(WorkerId, uv_loop_t*)> workerSetupCallback);
+            std::function<void(WorkerId, uv_loop_t*)> workerSetupCallback,
+            std::function<void()> exitCallback);
```

Comment:
>exitCallback [](start = 34, length = 12)

workerExitCallback is better name here. Also worker exitCallback may need exit status, yet I suggest not enhanced here. check in and fix it later.